tests/bdd/zfs.feature

```diff
@@ -0,0 +1,87 @@
+Feature: Validate volume provisioning for fsType zfs, ext4, xfs, btrfs
+         Validate snapshot creation and clone
+         Validate volume resize
+
+  Background:
+    Given a single zfs cluster is configured
+    And zfs pool pool created on the node
+
+  Scenario: volume provision for file system type zfs, ext4, xfs, btrfs
+    Given a storage class is created with fsType as zfs, ext4, xfs, btrfs respectively 
+    And pvc is created referencing the same sc
+    When a deployment is created using the same pvc
+    Then zfsvolume must be created and used by the application
```

Comment:
zfsvolume creation should not be dependent on any deployment creation using pvc. So we can try to write it in this way. WDYT ?

Given a storage class is created with fsType as <fsType>
And a pvc is created referencing this storage class
Then pvc should be in bound state 
And zfsvolume must be created and should be in Ready state to use by any application
When a application deployment is created using this pvc
Then deployment pod should be in Running state